### PR TITLE
Tag Flux.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [MikeInnes/Flux.jl](https://github.com/MikeInnes/Flux.jl)
Release: [v0.2.0](https://github.com/MikeInnes/Flux.jl/releases/tag/v0.2.0)
Diff: [vs v0.1.1](https://github.com/MikeInnes/Flux.jl/compare/854a1e18865742c59b6db6c58e16fee6ef9ef8ce...efcb9650da31c183b94b839f66aa3467d007c33f)
`requires` vs v0.1.1: 
```diff
--- 0.1.1/requires
+++ 0.2.0/requires
@@ -1,4 +1,4 @@
-julia 0.5
+julia 0.6
 Iterators
 DataFlow
 Juno
```
cc: @MikeInnes

### CODE REVIEW

**Comment by tkelman** [comment]
If you do `using MacroTools`, it needs to be listed in REQUIRE

File: Flux/versions/0.2.0/requires [hunk 1]
```diff
@@ -2,3 +2,4 @@ julia 0.6
 Iterators
 DataFlow
 Juno
+MacroTools
```
Comment (tkelman) [inline Flux/versions/0.2.0/requires]: 0.3.3 for `@q` https://github.com/MikeInnes/MacroTools.jl/commit/46e2c6f06073cc5fc708849f1a887d4b8a6f30c0

**Comment by tkelman** [comment]
Should not be extending base methods on base types as in `Base.squeeze(xs::Any)`

**Comment by MikeInnes** [comment]
Any idea why the tests are failing here?

**Comment by aviks** [comment]
I think we just branched. You probably got caught in the middle of it.

File: Flux/versions/0.2.0/requires [hunk 2]
```diff
@@ -0,0 +1,5 @@
+julia 00.6-
```
Comment (tkelman) [inline Flux/versions/0.2.0/requires]: what are you doing with the extra 0?

**Pull request merged**